src/site/layouts/news_story.drupal.liquid

```diff
@@ -0,0 +1,99 @@
+{% comment %}
+Example data:
+{
+    "entityId": "93",
+    "entityBundle": "news_story",
+    "entityPublished": false,
+    "promote": false,
+    "created": 1551049412,
+    "entityUrl": {
+        "breadcrumb": [
+            {
+                "url": {
+                "path": "/",
+                "routed": true
+            },
+                "text": "Home"
+            },
+            {
+                "url": {
+                "path": "/pittsburgh-health-care",
+                "routed": true
+            },
+                "text": "Pittsburgh Health Care System"
+            }
+            ],
+        "path": "/pittsburgh-health-care/news/opioid-use-review-clinic"
+    },
+    "title": "Opioid Use Review clinic",
+    "fieldAuthor": {
+        "entity": {
+            "title": "Sheila Tunney",
+            "fieldDescription": "Public Affairs Office"
+        }
+    },
+    "fieldImageCaption": "During a one-time, hourlong session, Opioid Use Review clinic staff talk with Veterans about their pain, discuss\r\nways to improve quality of life and provide opioid safety education, either in person in Pittsburgh or via\r\ntelehealth at regional VA outpatient clinics.",
+    "fieldIntroText": "Since 2014, more than 55 people* who may have otherwise overdosed and died have been saved with naloxone rescue kits issued to Veterans by VA Pittsburgh Healthcare System.",
+    "fieldMedia": {
+        "entity": {
+            "image": {
+                "alt": "Opioid Use Review clinic staff talk with Veterans about their pain",
+                "title": "",
+                "derivative": {
+                    "url": "http://stg.va.agile6.com/sites/default/files/styles/crop_2_1/public/2019-02/VAMC-Region-2019-02-21_0.jpg?itok=OwvOlLIz",
+                    "width": 468,
+                    "height": 326
+                }
+            }
+        }
+    },
+    "fieldFullStory": {
+        "processed": "<p>While that is an amazing fact, it is a small number compared to the more than 70,000* people in</p> ..."
+    }
+}
+{% endcomment %}
+{% if header == empty %}
+    {% assign header = "h4" %}
+{% endif %}
+{% include "src/site/includes/header.html" %}
+{% include "src/site/includes/preview-edit.drupal.liquid" %}
+{% include "src/site/includes/breadcrumbs.drupal.liquid" %}
+<div id="content" class="interior">
+    <main class="va-l-detail-page">
+        <div class="usa-grid usa-grid-full">
+            <nav class="usa-width-one-fourth va-sidebarnav" id="va-detailpage-sidebar"></nav>
```

Comment:
Do these pages not have a sidebar? Should we remove this empty element?

Reply:
they do have a sidebar... i just don't think it's made yet - part of another story. should i get rid of this for now?

Reply:
i removed this